src/features/click-to-play.js

```diff
@@ -2366,17 +2364,49 @@ const updateHandlers = {
             originalWindowDispatchEvent(new OriginalCustomEvent('ddg-ctp-youTubeVideoDetails', { detail: resp }))
         }
     },
-    enableSocialTracker: function (resp) {
-        originalWindowDispatchEvent(new OriginalCustomEvent('ddg-ctp-enableSocialTracker-complete', { detail: resp }))
+    unblockClickToLoadContent () {
+        originalWindowDispatchEvent(new OriginalCustomEvent('ddg-ctp-unblockClickToLoadContent-complete'))
```

Comment:
Note: I removed the `resp` parameter and stopped passing it through. There is no response and the `ddg-ctp-unblockClickToLoadContent-complete` event listener doesn't use it either.